bundles/org.openhab.binding.connectedcar/README.md

```diff
@@ -0,0 +1,585 @@
+| general      | lastUpdate                  | DateTime             | yes     | Last time data has been updated                                                                 |
+|              | lastAction                  | String               | yes     | Last action sent to the vehicle (check lastActionStatus for status/result)                      |
+|              | lastActionStatus            | String               | yes     | Result from last action sent to the vehicle                                                     |
+|              | lastActionPending           | Switch               | yes     | ON: An action was sent and is in status progressing                                             |
+|              | rateLimit                   | Number               | yes     | Number of remaining requests before the API gets throttled                                      |
+| control      | lock                        | Switch               | no      | Lock/Unlock doors                                                                               |
+|              | charge                      | Switch               | no      | Turn charger on/off                                                                             |
+|              | climater                    | Switch               | no      | Turn climatisation on/off                                                                       |
+|              | targetTemperature           | Number:Temperature   | yes     | Target temperature for the A/C climater                                                         |
+|              | targetChgLvl                | Number:Dimensionless | no      | CHarging stops automatically when the given level is reached                                    |
+|              | preHeater                   | Switch               | no      | Turn pre-heating on/off                                                                         |
+|              | duration                    | Number               | no      | Duration to run the ventilation/pre-heater in minutes                                           |
+|              | windowHeat                  | Switch               | no      | Turn window heating on/off                                                                      |
+|              | ventilation                 | Switch               | no      | Turn ventilation on/off                                                                         |
+|              | flash                       | Switch               | no      | ON: Triggers lights flashing                                                                    |
+|              | honkFlash                   | Switch               | no      | ON: Triggers Honk and Flash                                                                     |
```

Comment:
You could make these command channels. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#command-description